xml/System.Security.Cryptography/PbeParameters.xml

```diff
@@ -17,8 +17,9 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Represents parameters to be used for Password-Based Encryption (PBE).</summary>
     <remarks>To be added.</remarks>
+    <threadsafe>All public and protected members of <see cref="T:System.Security.Cryptography.PbeParameters" /> are thread-safe and may be used concurrently from multiple threads.</threadsafe>
```

Comment:
First time I see a `<threadsafe>` item in docs. Some questions:

1. Does it show up in IntelliSense?
2. If it does not, is it important enough to be moved to the summary, or should it be moved under remarks?

Reply:
As far as I know (I copied it from ConcurrentBag)

1) Yes
2) No
3) N/A (It's a distinct section on the docs page) -- https://docs.microsoft.com/en-us/dotnet/api/system.collections.concurrent.concurrentbag-1?view=netframework-4.8#thread-safety

Reply:
Yep, @bartonjs's correct

Reply:
Also, I think we have a request to bring the default one back when one is not provided like we used to have on MSDN